### PR TITLE
CNV- 20433: Loki logging queries

### PR DESCRIPTION
**Pull request by apinnick**
Version(s): 4.13

Issue: [CNV-20433](https://issues.redhat.com//browse/CNV-20433)

Link to docs preview: 
- [Viewing aggregated OpenShift Virtualization logs with the LokiStack](https://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#virt-viewing-logs-loki_virt-troubleshooting) - QE review required
- [Events](https://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#events_virt-troubleshooting) - Formatting changes. No QE review required
- [Viewing OpenShift Virtualization pod logs](https://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#virt-viewing-logs-cli_virt-troubleshooting) - Procedure split into 2 modules. QE review required to make sure I didn't introduce errors.

QE review:
- [ ] QE has approved this change.

Additional information: 

This PR includes some doc improvements. 

I am requesting peer review at the same time as QE review because I need to work on the RN PR, which needs to link to the new sections.

### CODE REVIEW

File: modules/virt-viewing-logs-loki.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+:_content-type: PROCEDURE
+[id="virt-viewing-logs-loki_{context}"]
+= Viewing aggregated {VirtProductName} logs with the LokiStack
+
+You can view aggregated logs for {VirtProductName} pods and containers by using the LokiStack in the web console.
+
+.Prerequisites
+
+* You deployed the LokiStack.
```
Comment (sradco) [inline modules/virt-viewing-logs-loki.adoc:13]: @apinnick  should we add a link to the Loki installation guide here?

Comment (apinnick) [reply modules/virt-viewing-logs-loki.adoc:13]: No. That is outside the scope of this task. In the "Additional resources" section, there is a [link to Deploying LokiStack on OCP](https://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#additional-resources_virt-troubleshootinghttps://58993--docspreview.netlify.app/openshift-enterprise/latest/virt/support/virt-troubleshooting.html#additional-resources_virt-troubleshooting).

File: modules/virt-configuring-pod-log-verbosity.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.Procedure
+
+. To set log verbosity for specific components, open the `HyperConverged` CR in your default text editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Set the log level for one or more components by editing the `spec.logVerbosityConfig` stanza. For example:
```
Comment (kelbrown20) [inline modules/virt-configuring-pod-log-verbosity.adoc:20]:
```suggestion
. Set the log level for one or more components by editing the `spec.logVerbosityConfig` path. For example:
```

File: virt/support/virt-troubleshooting.adoc [hunk 3]
```diff
@@ -34,13 +40,38 @@ $ oc describe <resource> <resource_name>
...
+
+include::modules/virt-loki-log-queries.adoc[leveloffset=+3]
+
 
-include::modules/virt-common-error-messages.adoc[leveloffset=+2]
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+==== Additional resources for LokiStack and LogQL
+* xref:../../logging/cluster-logging-loki.adoc#about-logging-loki_cluster-logging-loki[About the LokiStack]
+* xref:../../logging/cluster-logging-loki.adoclogging-loki-deploy_cluster-logging-loki[Deploying the LokiStack] on {product-title}
```
Comment (kelbrown20) [inline virt/support/virt-troubleshooting.adoc:73]: Missing the `#`
```suggestion
* xref:../../logging/cluster-logging-loki.adoc#logging-loki-deploy_cluster-logging-loki[Deploying the LokiStack on {product-title}]
```

**Review by kelbrown20 — COMMENTED** [review]
Hello Avital! This is fantastic work! I did not have much to add other than some wording nits and fixing some issues with links. Great work!

File: modules/virt-configuring-pod-log-verbosity.adoc (hunk 2 above)
Comment (apinnick) [reply modules/virt-configuring-pod-log-verbosity.adoc:20]: This is incorrect. The correct term for a section in a YAML file is "stanza", not "path".

File: modules/virt-viewing-logs-cli.adoc [hunk 4]
```diff
@@ -4,40 +4,12 @@
 
 :_content-type: PROCEDURE
 [id="virt-viewing-logs-cli_{context}"]
-= Viewing {VirtProductName} logs with the CLI
+= Viewing {VirtProductName} pod logs with the CLI
 
-You can configure the verbosity level of {VirtProductName} component logs by editing the `HyperConverged` custom resource (CR). Then, you can view logs for the component pods by using the `oc` CLI tool.
+You can view logs for the {VirtProductName} pods by using the `oc` CLI tool.
```
[the remark this answers is not in the document]
Comment (apinnick) [reply modules/virt-viewing-logs-cli.adoc:9]: This is incorrect. See [IBM Style Guide usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#u).

**Comment by stesrn** [comment]
@apinnick change looks good

**Pull request merged**